flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,485 @@
+        out_list = dict()
+        out_list['extra'] = True
+        if ep_nr is not None:
+            out_list['ep'] = ep_nr
+            out_list['extra'] = False
+        if file_version:
+            out_list['version'] = file_version
+        if is_special:
+            out_list['special'] = is_special
+        if is_op_ed:
+            out_list['op-ending'] = is_op_ed
+        if is_trailer:
+            out_list['trailer-promo'] = is_trailer
+        return out_list
+
+    def parse_from_xml(self, xml_entries, config):
+        entries = []
+        for xml_entry in xml_entries:
+            new_entry = Entry()
+            # skip if we have no link/title
+            if not xml_entry.title or not xml_entry.link:
+                continue
+            # fill base data
+            new_entry['title'] = xml_entry.title  # needs to-be fixed manually later
+            new_entry['url'] = xml_entry.link  # file link
+            episode_data = self.parse_episode_data('title', new_entry)
+            if episode_data['extra'] is True:
+                if 'include' in config:
+                    if not any(key in episode_data for key in config['include']):
+                        continue  # is extra file
+                else:
+                    continue
+            # store some usefully data in the namespace
+            fid_string = self.get_value_via_regex('url', new_entry, r'/f([0-9]{1,10})$')
+            if fid_string:
+                try:
+                    fid = int(fid_string)
+                    if fid is not None:
+                        new_entry[NAMESPACE_PREFIX_MAIN + 'fid'] = fid
+                except (TypeError, ValueError) as ex:
+                    log.error('Skipping Entry: %s, could not parse fid string: %s, error: %s', new_entry, fid_string, ex)
+                    continue
+            if xml_entry.updated_parsed:
+                new_entry['rss_pubdate'] = datetime.fromtimestamp(mktime(xml_entry.updated_parsed))
+            elif xml_entry.updated:
+                parsed_date = convert_to_naive_utc(xml_entry.updated)
+                if parsed_date:
+                    new_entry['rss_pubdate'] = parsed_date
+
+            # add some usefully attributes to the namespace
+            self._fill_namespace_attributes(xml_entry, new_entry)
+            if config['valid_only'] is True:
+                if self.is_valid_entry(new_entry, NAMESPACE_PREFIX) is False:
+                    continue
+            # now manually fix known fields
+            # 344.18 MB (360.895.922)
+            size_string = self.get_value_via_regex(NAMESPACE_PREFIX + 'size', new_entry, r'\((([0-9]{1,3}\.|[0-9]{1,3}){1,5})\)')
+            if size_string:
+                bytes_string = size_string.replace('.', '')
+                try:
+                    size_bytes = int(bytes_string)
+                    size_mb = int(size_bytes / 1024 / 1024)  # MB
```

Comment:
Kinda, but i want to store the org 'safe' size in bytes anyway. Using the byte string at the end of the input string seemed safer to me, than relying on proper MB/GB/KB tokens.
Is there a version that only parses valid byte like numbers?